couchdb.go

```diff
@@ -279,3 +282,12 @@ func (db *DB) SyncDesign(d *Design) error {
 	}
 	return nil
 }
+
+func errorGetByEmptyID() error {
+	return &Error{
```

Comment:
What was the objective of faking a CouchDB response as opposed to raising an invalid argument error? Please correct me if you think different, but an empty ID should be checked for by the code calling `Get`.

Reply:
Requesting a document with an invalid ID shouldn't return a successful response in any case, that only creates unexpected behaviours.

Yes, client calling `Get` should have checked the ID, but it may not and bugs keep reproducing from here (because this one returns another document with empty values, you can read a property from that document and call `Get` again and again, and in each query there's no real document involved, just fake responses from here)